src/pages/patterns/dialog-pattern/index.mdx

```diff
@@ -188,6 +188,8 @@ Use modeless dialogs to display non-critical information or optional tasks relat
 
 Use when a user needs to compare or refer to information in the main page alongside the modal. users can interact with the modeless content and the on-page content simultaneously.
 
+A modeless dialog window may be moved from its original placement on the screen. This allows the user to access information that might otherwise be covered up by the dialog.
```

Comment:
```suggestion
A modeless dialog window can be moved from its original placement on the screen. This allows the user to access information that might otherwise be covered up by the dialog.
```